docs/LogGuide.md

```diff
@@ -845,39 +1306,145 @@ Marker Code | Name | Sample Locations | Consistent meaning?
+
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=HeadMarker&lang=en) -->
+
+#### Structure
+
+```log
+Network Log Line Structure:
+27|2021-04-26T14:11:35.0000000-04:00|[targetId]|[target]|||[id]
+
+ACT Log Line Structure:
+[14:11:35.0] 1B:27:2021-04-26T14:11:35.0000000-04:00:[targetId]:[target]:::[id]
+```
+
+#### Regexes
+
+```log
+Network Log Line Regex:
+^(?<type>(?:27))\|(?<timestamp>(?:[^|]*))\|(?<targetId>(?:[^|]*))\|(?<target>(?:[^|]*))\|(?:[^|]*\|){2}(?<id>(?:[^|]*))\|
+
+ACT Log Line Regex:
+(?<timestamp>(?:^.{14})) 1B:(?<targetId>(?:[0-9A-F]{8})):(?<target>(?:[^:]*?)):....:....:(?<id>(?:....)):
+```
+
+#### Examples
+
+```log
+Network Log Line Examples:
+27|2021-04-26T14:37:48.3550000-04:00|10FF0001|Tini Poutini|0000|0000|0018|0000|0000|0000||051a1ab18b11a70c22fe56d716a75a20
+27|2021-04-26T14:38:00.0360000-04:00|10FF0001|Tini Poutini|0000|0000|001A|0000|0000|0000||81351db91d8a53406106f4fe94dccf3a
+27|2021-04-26T14:29:35.7180000-04:00|10FF0001|Tini Poutini|0000|E000|00A0|0000|0000|0000||44b502d374749978aa96969869b49afc
+27|2021-04-26T14:17:31.6980000-04:00|10FF0001|Tini Poutini|0000|A9B9|0057|0000|0000|0000||4fb326d8899ffbd4cbfeb29bbc3080f8
+27|2021-05-11T13:48:45.3370000-04:00|40000950|Copied Knave|0000|0000|0117|0000|0000|0000||fa2e93fccf397a41aac73a3a38aa7410
+
+ACT Log Line Examples:
+[14:37:48.355] 1B:27:2021-04-26T14:37:48.3550000-04:00:10FF0001:Tini Poutini:0000:0000:0018:0000:0000:0000::051a1ab18b11a70c22fe56d716a75a20
+[14:38:00.36] 1B:27:2021-04-26T14:38:00.0360000-04:00:10FF0001:Tini Poutini:0000:0000:001A:0000:0000:0000::81351db91d8a53406106f4fe94dccf3a
+[14:29:35.718] 1B:27:2021-04-26T14:29:35.7180000-04:00:10FF0001:Tini Poutini:0000:E000:00A0:0000:0000:0000::44b502d374749978aa96969869b49afc
+[14:17:31.698] 1B:27:2021-04-26T14:17:31.6980000-04:00:10FF0001:Tini Poutini:0000:A9B9:0057:0000:0000:0000::4fb326d8899ffbd4cbfeb29bbc3080f8
+[13:48:45.337] 1B:27:2021-05-11T13:48:45.3370000-04:00:40000950:Copied Knave:0000:0000:0117:0000:0000:0000::fa2e93fccf397a41aac73a3a38aa7410
+```
+
+<!-- AUTO-GENERATED-CONTENT:END -->
 
 ### 1D: NetworkTargetMarker
 
-Unknown?
+This message indicates a target marker placed above or removed from a combatant's head by a player.
+
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=networkTargetMarker&lang=en) -->
+
+#### Structure
+
+```log
+Network Log Line Structure:
+29|2021-04-26T14:11:35.0000000-04:00|[operation]|[waymark]|[id]|[name]|[targetId]|[targetName]
+
+ACT Log Line Structure:
+[14:11:35.0] 1D:29:2021-04-26T14:11:35.0000000-04:00:[operation]:[waymark]:[id]:[name]:[targetId]:[targetName]
+```
+
+#### Examples
+
+```log
+Network Log Line Examples:
+29|2021-06-10T20:15:15.1000000-04:00|Delete|0|10FF0001|Tini Poutini|4000641D|||50460af5ff3f8ec9ad03e6953d3d1ba9
+29|2021-06-10T20:15:15.1000000-04:00|Add|0|10FF0001|Tini Poutini|400075A9|||0afeb301004fc6d8c7bd5f6cbda9dce7
+29|2021-05-25T22:54:32.5660000-04:00|Add|6|10FF0001|Tini Poutini|10FF0002|Potato Chippy||70a8c8a728d09af83e0a486e8271cc57
```

Comment:
I think a Delete and Add line are plenty, and we don't need two Adds.

I think also you should probably describe what operation and waymark can be here if you're going to document this log line? (or leave a TODO)?

Reply:
Maybe drop the middle example, so we have an example with ID 0 and an example with ID 6 instead of two 0s?